README.md

```diff
@@ -1,13 +1,65 @@
 # Humanitec Google Cloud Reference Architecture Implementation
+> TL;DR
+>
+> [Follow this tutorial to spin up your Humanitec Google Cloud Reference Architecture Implementation](README.md#how-to-spin-up-your-humanitec-google-cloud-reference-architecture-implementation)
```

Comment:
Make it more clear that we are staying in the _same_ tutorial here, like "Skip the theory and go straight to spinning up your Humanitec Google Cloud Reference Architecture Implementation".